llm_bench/python/utils/nncf_utils.py

```diff
@@ -35,45 +35,3 @@
 
 def get_compressed_path(output_dir: str, base_precision, option: str):
     return Path(output_dir) / "pytorch/dldt/compressed_weights" / f"OV_{base_precision}-{option}"
-
-
-INT4_MODEL_CONFIGURATION = {
```

Comment:
Yet need to resolve some config misallignments:

- For `databricks/dolly-v2-3b` in optimum `ratio` is 1 while in genai is 0.8.

- Some configs from genai are missing in optimum:
```
red-pajama-incite-7b-instruct
llama-2-7b
llama-2-7b-chat
llama-2-13b-chat
stablelm-epoch-3b-preview
stable-zephyr-3b-dpo
rocket-3b
qwen-7b-chat
open-llama-3b
orca-mini-3b
mixtral-8x7b-v0.1
baichuan2-7b-chat
mistral-7b-v0.1
llama-7b
red-pajama-incite-chat-3b-v1
longchat-b7
tiny-llama-1.1b-chat
```

- Some configs in optimum are missing in genai:
```
togethercomputer/RedPajama-INCITE-7B-Instruct
meta-llama/Llama-2-7b
meta-llama/Llama-2-7b-chat
meta-llama/Llama-2-13b-chat
stabilityai/stablelm-zephyr-3b
pansophic/rocket-3B
Qwen/Qwen-7B-Chat
openlm-research/open_llama_3b
openlm-research/open_llama_3b_v2
psmathur/orca_mini_3b
mistralai/Mixtral-8x7B-v0.1
togethercomputer/RedPajama-INCITE-Chat-3B-v1
mistralai/Mistral-7B-v0.1
baichuan-inc/Baichuan2-7B-Chat
openai-community/gpt2
lmsys/longchat-7b-16k
TinyLlama/TinyLlama-1.1B-Chat-v1.0
stabilityai/stablelm-tuned-alpha-7b
```

Reply:
Resolved